lib/internal/util/parse_args/parse_args.js

```diff
@@ -304,13 +304,51 @@ function argsToTokens(args, options) {
   return tokens;
 }
 
+/**
+ * Format help text for printing.
+ * @param {string} longOption - long option name e.g. 'foo'
+ * @param {object} optionConfig - option config from parseArgs({ options })
+ * @returns {string} formatted help text for printing
+ * @example
+ * formatHelpTextForPrint('foo', { type: 'string', help: 'help text' })
+ * // returns '--foo <arg>                   help text'
+ */
+function formatHelpTextForPrint(longOption, optionConfig) {
+  const layoutSpacing = 30;
+
+  const shortOption = objectGetOwn(optionConfig, 'short');
+  const type = objectGetOwn(optionConfig, 'type');
+  const help = objectGetOwn(optionConfig, 'help');
+
+  let helpTextForPrint = '';
+  if (help) {
```

Comment:
I suggest that options without explicit help text are still included in the formatted help. This makes basic help simple. Add any help field, including describing the overall program, and you get formatted help returned with all the available options included.

Yes, some users will want "hidden" options. But I think that is uncommon, and more useful to make it easy to get help text for simple programs with (well named) options, without needing to describe each option.

Reply:
(`--charlie` and `--india` in the big example in the tests.)

Reply:
It's also a lot easier to filter out help text you don't want, than to recreate and format help text you do want.

Reply:
I was imagining that supplying any `help` text, whether the general `help` or for an option, would trigger returning the formatted help in the results. The current implementation requires passing a general `text` property. (And the documentation says that too, so self consistent.)

I don't feel strongly about this. See what others think.

If the help without general text was allowed it would look like:
```
% node index.cjs --help 
-v, --verbose
-h, --help                    Prints command line options
--output <arg>                Output directory
```

Reply:
Got it, thanks for clarifying!